libnmstate/schema.py

```diff
@@ -41,9 +41,22 @@ class Interface(object):
     MTU = 'mtu'
 
 
+class LinuxRoute(object):
```

Comment:
Any reason you prefixed it with `Linux`?

Reply:
Changed to `Route`.